### PR TITLE
Add link to download official PSD files

### PR DESCRIPTION
**Pull request by weizmannk**
The old link to the PSD files on LIGO DCC seems not to work ,
this new link  works ok.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -49,7 +49,7 @@ O6-psds = \
 #
 
 %.txt:
-	curl -L https://dcc.ligo.org/LIGO-T2000012/public/$(@F) > $@
+	curl -L https://dcc.ligo.org/public/0165/T2000012/001/$(@F) > $@
```
Comment (lpsinger) [inline Makefile:52]:
```suggestion
	curl -L https://dcc.ligo.org/LIGO-T2000012-v1/public/$(@F) > $@
```

File: pack-psds.py [hunk 2]
```diff
@@ -38,7 +38,7 @@
 
     fgrid = np.arange(f0, fmax, df)
     series = lal.CreateREAL8FrequencySeries(
-        psd_file.name, 0, f0, df, lal.SecondUnit, len(fgrid))
+        (psd_file.name).split(".")[0], 0, f0, df, lal.SecondUnit, len(fgrid))
```
Comment (lpsinger) [inline pack-psds.py:41]: Why are the PSD entries renamed?

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: pack-psds.py (hunk 2 above)
Comment (weizmannk) [reply pack-psds.py:41]: I just remove the extension ".txt", 
the extension generates raises errors 
like this :

  File "/home/weizmann.kiendrebeogo/anaconda3/envs/observing_scenarios/lib/python3.9/site-packages/ligo/lw/ligolw.py", line 472, in enc
    raise ValueError("invalid Name '%s'" % name)
ValueError: invalid Name '****avirgo_O3actual.txt'****

After remove the extension the psds name  become avirgo_O3actual..... without  the extension.
and this work .

**Review by lpsinger — APPROVED** [review]

File: pack-psds.py (hunk 2 above)
Comment (lpsinger) [reply pack-psds.py:41]: Thanks. I was able to reproduce locally after updating python-ligo-lw.

**Pull request merged**